docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -31,6 +80,11 @@ local common = import 'common.libsonnet';
       },
     }],
   }],
+  filePool: {
+    blockDevice: {
+      devicePath: '/worker/filepool_cache',
```

Comment:
Nit: my recommendation is to remove the _cache suffix here. The file pool is just a short term temporary file store. It isn’t really a cache.